pkg/operator/collection.go

```diff
@@ -371,12 +371,6 @@ func (r *collectionReconciler) makeCollectorConfig(ctx context.Context, spec *mo
 		cfgs, err := pmon.ScrapeConfigs(projectID, location, cluster)
 		if err != nil {
 			msg := "generating scrape config failed for PodMonitoring endpoint"
-			cond = &monitoringv1.MonitoringCondition{
```

Comment:
I actually think this is a bug. We never update the status on a bad config.

Can you add a TODO to fix with your deletion?
```go
// TODO: surface ScrapeConfigError to PodMonitoring status here.
```